### DISCUSSION TITLE
need solution

### DISCUSSION THREAD

**Discussion by vytska69** [ROM-builders/temporary#9574] [General]
i'm trying to build DotOS12.1, but test script failing. @Apon77 can you look at my branch? my last build: https://cirrus-ci.com/build/6142032841277440 
thanks. ;)

**Comment by Apon77** [comment]
> i'm trying to build DotOS12.1, but test script failing. @Apon77 can you look at my branch? my last build: https://cirrus-ci.com/build/6142032841277440 thanks. ;)

fix branch name in init line

**Reply by vytska69** [reply]
okay, i'll look
